packages/fxa-shared/package.json

```diff
@@ -54,9 +54,10 @@
   },
   "scripts": {
     "postinstall": "yarn build || true",
-    "build": "tsc --build && yarn copy-assets",
-    "compile": "tsc --noEmit",
+    "build": "tsc --build && yarn copy-assets && yarn copy-sql",
+    "compile": "tsc --noEmit && yarn copy-assets && yarn copy-sql",
     "copy-assets": "cp -r ./db/luaScripts ./dist/db/luaScripts",
+    "copy-sql": "find test -name \\*.sql -exec cp \\{\\} dist/\\{\\} \\;",
```

Comment:
This is probably because I haven't reviewed any other token audit PRs but why this change in `fxa-shared`?

Reply:
I was amazed this hadn't caused a problem previously, but without this the the dist folder wouldn't contain `.sql` files, and some of the code expects those files to be there. This [line](https://github.com/mozilla/fxa/blob/c3de9f95e2cd92694039782415248c23d1f66874/packages/fxa-shared/test/db/models/profile/helpers.ts#L15) would consistently fail, without this addition.

The issue crept into the admin server, because we now compile scripts folder. We need to do this, so that SRE can run the script with node sans any ts compilation. The `admin-server/scripts/email-bounct.ts` [file](https://github.com/mozilla/fxa/blob/c3de9f95e2cd92694039782415248c23d1f66874/packages/fxa-admin-server/scripts/email-bounce.ts#L13) references fxa-shared/test/db/models/profile/helpers.ts, which is how that got tangled up and ultimately resulted in a build error.

Anyways, I think the `.sql` files should end up in the dist folder if the compiled code anticipates they are there.